plugins/inventory/lxd.py

```diff
@@ -0,0 +1,1060 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Frank Dornheim <dornheim@posteo.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: lxd
+    plugin_type: inventory
+    short_description: Returns Ansible inventory from lxd host
+    description:
+        - Get inventory from the lxd
+        - Uses a YAML configuration file that ends with 'lxd.(yml|yaml)'.
+    version_added: "2.1.0"
```

Comment:
2.1.0 has already been released. The next expected release is 2.3.0:
```suggestion
    version_added: "2.3.0"
```

Reply:
done